src/main/java/org/aksw/simba/squirrel/frontier/Frontier.java

```diff
@@ -68,4 +69,18 @@
      */
     int getNumberOfPendingUris();
 
+
+    /**
+     * Indicates whether this instance does recrawling, which means that it keeps already crawled uris, and crawls
+     * them again at a specified date.
+     *
+     * @return True iff recrawling is active.
+     */
+    boolean doesRecrawling();
+
+    /**
+     * Close everything connected to this frontier (e.g. timers).
+     */
+    void shutdown();
```

Comment:
It might be better to let the `Frontier` interface extend `Closeable` or `AutoCloseable`.